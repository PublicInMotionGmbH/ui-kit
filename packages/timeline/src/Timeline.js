import React from 'react'
import PropTypes from 'prop-types'

import { buildClassName } from '@talixo/shared'

const moduleName = 'timeline'

const propTypes = {
  /** Additional class name */
  className: PropTypes.string
}

const defaultProps = {
}

/**
 * Component which represents Timeline.
 *
 * @param {object} props
 * @param {string} [props.className]
 * @returns {React.Element}
 */
function Timeline (props) {
  const { className, ...passedProps } = props
  const clsName = buildClassName(moduleName, className)

  return (
    <div className={clsName} {...passedProps} >
      {/* {children} */}
    </div>
  )
}

Timeline.displayName = 'Timeline'

Timeline.propTypes = propTypes
Timeline.defaultProps = defaultProps

export default Timeline
