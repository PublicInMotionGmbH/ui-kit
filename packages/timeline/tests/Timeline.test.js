import React from 'react'
import { shallow } from 'enzyme'

import Timeline from '../src/Timeline'

describe('<Timeline />', () => {
  it('renders children correctly', () => {
    const wrapper = shallow(<Timeline />)

    expect(wrapper).toMatchSnapshot()
  })
})
