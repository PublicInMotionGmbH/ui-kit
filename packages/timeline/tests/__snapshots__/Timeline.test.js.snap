// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timeline /> renders children correctly 1`] = `
<span
  className="talixo-timeline"
/>
`;
